src/web/components/CMP.test.tsx

```diff
@@ -35,24 +36,23 @@ describe('CMP', () => {
     it('It should render null if shouldShow returns false', () => {
         shouldShow.mockImplementation(() => false);
 
-        const { container } = render(<CMP cmpUi={true} />);
+        const { container } = render(<CMP />);
 
         expect(container.firstChild).toBeNull();
     });
 
     it('It should render null if shouldShow ireturns true but cmpUi switch is false', () => {
```

Comment:
This test implies the component should not render if `cmpUI` is false

Reply:
yes I should remove that

Reply:
I'd prefer we keep the test to make sure we're doing things right and also as a form of documentation